ocw/lib/gce.py

```diff
@@ -106,77 +93,54 @@ def get_error_reason(error: "googleapiclient.errors.HttpError") -> str:
             pass
         return reason
 
-    def cleanup_all(self):
+    def cleanup_all(self) -> None:
         self.log_info("Call cleanup_all")
 
         self.log_dbg("Disks cleanup")
         for region in self.list_regions():
             for zone in self.list_zones(region):
                 self.log_dbg("Searching for disks in {}", zone)
-                request = self.compute_client().disks().list(project=self.project, zone=zone)
-                while request is not None:
-                    response = request.execute()
-                    if "items" not in response:
-                        self.log_dbg("response has no items. id={}", response["id"])
-                        break
-                    self.log_dbg("response has {} items. id={}", len(response["items"]), response["id"])
-                    for disk in response["items"]:
-                        if self.is_outdated(parse(disk["creationTimestamp"]).astimezone(timezone.utc)):
-                            if self.dry_run:
-                                self.log_info("Deletion of disk {} created on {} skipped due to dry run mode",
-                                              disk["name"], disk["creationTimestamp"])
-                            else:
-                                delete_request = (
-                                    self.compute_client()
-                                    .disks()
-                                    .delete(project=self.project, zone=zone, disk=disk["name"])
-                                )
-                                try:
-                                    delete_response = delete_request.execute()
-                                    self.log_dbg("Deletion response: {}", delete_response)
-                                    self.log_info("Disk '{}' deleted", disk["name"])
-                                except HttpError as err:
-                                    if GCE.get_error_reason(err) == 'resourceInUseByAnotherResource':
-                                        self.log_dbg("Disk {} can not be deleted because in use", disk["name"])
-                                    else:
-                                        raise err
-                    request = (
-                        self.compute_client()
-                        .disks()
-                        .list_next(previous_request=request, previous_response=response)
-                    )
+                disks = self._paginated(self.compute_client().disks, project=self.project, zone=zone)
```

Comment:
you lost debug message here `self.log_dbg("response has no items. id={}", response["id"])` . you still can add something like : 
`self.log_dbg("{} disks found", len(disks))`